submissions/Kostiantyn_Rakhmanov/frogger/app.js

```diff
@@ -0,0 +1,85 @@
+ let Enemy = function(x, y, speed) {
+    this.x = x;
+    this.y = y;
+    this.speed = speed;
+    this.sprite = 'images/enemy-bug.png';
+};
+
+Enemy.prototype.update = function(dt) {
+    this.x += this.speed * dt;
+
+    if (this.x > 510) {
+        this.x = -100;
```

Comment:
Mystery numbers item 3 of the requirements.
Here and across the entire codebase.